server/src/test/java/com/broadcom/lsp/cobol/service/CobolTextDocumentServiceTest.java

```diff
@@ -325,6 +327,26 @@ void testDidCloseDisposeDiagnostics() {
         .publishDiagnostics(ImmutableMap.of(UseCaseUtils.DOCUMENT_URI, ImmutableList.of()));
   }
 
+  /**
+   * This test verify that when a {@link
+   * CobolTextDocumentService#ranges(JsonObject)} is sent from the client to
+   * get document unit ranges, it processes properly.
+   */
+  @Test
+  void testRanges() throws ExecutionException, InterruptedException {
+    String uri = UUID.randomUUID().toString();
```

Comment:
Please, use some meaningful data for tests. I understand that it doesn't make sense for tests, but it helps a developer to understand what kind of data used, in what format, etc.

Reply:
Fixed